PROCESSORS.md

```diff
@@ -1448,6 +1449,60 @@ In the list below, the names of required properties appear in bold. Any other pr
 |success|Files that have been successfully written to Azure storage are transferred to this relationship|
 
 
+## PutGcsObject
+
+### Description
+
+Puts content into a Google Cloud Storage bucket
+### Properties
+
+In the list below, the names of required properties appear in bold. Any other properties (not in bold) are considered optional. The table also indicates any default values, and whether a property supports the NiFi Expression Language.
+
+| Name                                  | Default Value | Allowable Values                                                                                          | Description                                                                                                                                                                                                                                                                                       |
+|---------------------------------------|---------------|-----------------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **Bucket Name**                       |               |                                                                                                           | The name of the Bucket to upload to. If left empty the _gcs.bucket_ attribute will be used by default.<br>**Supports Expression Language: true**                                                                                                                                                  |
+| **Object Name**                       |               |                                                                                                           | The name of the object to be uploaded. If left empty the _filename_ attribute will be used by default.<br>**Supports Expression Language: true**                                                                                                                                                  |
```

Comment:
afaik we cant extract the flowfile attributes via expression language like Nifi can so we cant really use the same interface
Or are you referring to use **Key** instead of **Object Name**?

Reply:
I think we can extract the attributes via the expression language. I just want to aim to be compatible if there is no reason not to be.

Reply:
You are right, I dont know why I thought that.
Reworked the properties so they are more consistent with nifi's version
https://github.com/apache/nifi-minifi-cpp/pull/1268/commits/57404afcf541daa1780365e054a555dcd8093c3e